packages/framework/src/Resources/views/Admin/Form/theme.html.twig

```diff
@@ -269,22 +269,32 @@
             {% if js_container is not null %}
                 <div class="{{ js_container.container_class }}" data-type="{{ js_container.data_type }}">
             {% endif %}
-            <div class="form-line">
+            <div class="form-line{% if multidomain is defined and isMultidomain() %} form-line--no-horizontal-gap{% endif %}">
             {% if multidomain is defined %}
                 {% if isMultidomain() %}
                     {{ form_label(form, getDomainName(domainId)) }}
-                    {{ domainIcon(domainId) }}
                 {% endif %}
             {% else %}
                 {{ form_label(form, label) }}
             {% endif %}
             {{- form_errors(form) -}}
             {% if newline is not defined %}
-                <div{% if isMultidomain() %} class="form-line__side"{% endif %}>
-                    <div class="form-line__item form-line__item--full-width">
-                        {{- form_widget(form) -}}
+                {% if isMultidomain() %}
+                    <div class="form-line__side form-line__side--multidomain">
+                        <div class="input__wrap">
+                            {{- form_widget(form, { attr: { class: 'input--domain' }}) -}}
+                            <span class="input__domain">
+                                {{ domainIcon(domainId) }}
+                            </span>
+                        </div>
                     </div>
-                </div>
+                {% else %}
+                    <div>
```

Comment:
It is necessary this div without any class?

Reply:
Fixed